contrib_bots/bots/converter/converter.py

```diff
@@ -48,86 +49,108 @@ def usage(self):
                '''
 
     def triage_message(self, message, client):
-        return '@convert' in message['content']
-
-    def handle_message(self, message, client, state_handler):
-        content = message['content']
+        # To prevent infinite loop in private message, bot will detect
```

Comment:
`message` should be plural here

A bit of rephrasing, maybe `bot will return false if the sender name is the same as the bot name`?